src/app/components/elements/list-groups/ListGroupFooterBottom.tsx

```diff
@@ -1,37 +1,69 @@
 import React from "react";
 import { ListGroup, ListGroupItem } from "reactstrap";
 import { ExternalLink } from "../ExternalLink";
+import { Link } from "react-router-dom";
+interface FooterLinkProps {
+  linkTo: string;
+  children?: React.ReactNode | string;
+}
+
+const FooterLink = ({ linkTo, children }: FooterLinkProps) => {
+  return (
+    <ListGroupItem className="border-0 px-0 py-0 bg-transparent align-items-stretch">
+      <Link className="footerLink" to={linkTo}>
+        {children}
+      </Link>
+    </ListGroupItem>
+  );
+};
+
+const footerLinks = {
+  right: [
+    <FooterLink key="accessibility-footer-link" linkTo="/accessibility">
+      Accessibility
+    </FooterLink>,
+    <FooterLink key="cookie-policy-footer-link" linkTo="/cookies">
+      Cookie policy
+    </FooterLink>,
+    <FooterLink key="privacy-policy-footer-link" linkTo="/privacy">
+      Privacy policy
+    </FooterLink>,
+    <FooterLink key="terms-of-use-footer-link" linkTo="/terms">
+      Terms of use
+    </FooterLink>,
+  ],
+};
 
 export const ListGroupFooterBottom = () => (
   <div className="footer-links footer-bottom">
-    <ListGroup className="d-flex flex-wrap flex-row">
+    <ListGroup className="d-flex flex-wrap flex-row link-group">
+      <ListGroupItem className="footer-bottom-links d-md-flex flex-md-row bg-transparent link-group w-100">
```

Comment:
could we just check the horizontal padding on this one please ("Links"), it's not quite aligned compared to the rest of the content. Looks to be 4px too much or so: 

<img width="795" alt="image" src="https://github.com/user-attachments/assets/004aba95-d9f9-46ce-8d25-477554d5d287">